### PR TITLE
VACMS-1191: New and refactored path patterns

### PR DESCRIPTION
**Pull request by oksana-c**
## Description

See #1191 , #1328. 

- Added path pattern for banners.
- Deleted duplicate pattern for events.
- Added pattern for VAMC system health services & facility health service
- Normalized path patterns naming to match content types.
- Added hook_update() to regenerate aliases for select CTs. Also included `event` CT here - see #1328 

## Testing done

Visual

## QA steps

1. Review path patterns at /admin/config/search/path/patterns
1. Check if aliases are generated according to defined patterns /admin/config/search/path/patterns by  filtering Content view by each of targeted content types (Event, VAMC system banner alert with situation updates, VAMC facility health service, VAMC system health service, Publication, Support Service) 
1. View a node from each of targeted content types and verify that new revision was not created when the node was saved via hook_update

### CODE REVIEW

File: docroot/modules/custom/va_gov_backend/va_gov_backend.install [hunk 1]
```diff
@@ -226,3 +226,45 @@ function va_gov_backend_update_8004() {
...
+ */
+function va_gov_backend_update_8005() {
+  $updated = 0;
+  $failed = 0;
+  $failed_nids = [];
+
+  // Adding event ct here, to cover any aliases from VACMS-1328.
+  $bundles = [
+    'event',
+    'full_width_banner_alert',
```
Comment (swirtSJW) [inline docroot/modules/custom/va_gov_backend/va_gov_backend.install]: I don't think we want to trigger a save to existing banner alerts saves are capable of triggering a send to GovDelivery which can trigger a lot of repeat email to people.   Is there an essential reason to update banner alert aliases?

File: docroot/modules/custom/va_gov_backend/va_gov_backend.install [hunk 2]
```diff
@@ -226,3 +226,45 @@ function va_gov_backend_update_8004() {
...
+
+    Drupal::logger('va_gov_backend')
+      ->log(LogLevel::INFO, 'Path Alias for %bundle: Successfully updated path alias for %updated out of %count entities. Failed to update %failed out of %count. %failed_nids', [
+        '%bundle' => $bundle,
+        '%count' => count($entities),
+        '%updated' => $updated,
+        '%failed' => $failed,
+        '%failed_nids' => 'Failed nids: ' . implode(', ', $failed_nids) . '.',
+      ]);
+  }
```
Comment (swirtSJW) [inline docroot/modules/custom/va_gov_backend/va_gov_backend.install]: Generally what kind of numbers are we expecting here? Less than 50, less than 100, less than 1000...?

Comment (oksana-c) [reply docroot/modules/custom/va_gov_backend/va_gov_backend.install]: not much - total 545 entities, that's why didn't batch them

snippets from log:
- Path Alias for support_service: Successfully updated path alias for 16 out of 16 entities.
- Path Alias for regional_health_care_service_des: Successfully updated path alias for 123 out of 123 entities.
- Path Alias for outreach_asset: Successfully updated path alias for 177 out of 177 entities.
- Path Alias for health_care_local_health_service: Successfully updated path alias for 157 out of 157 entities.
- Path Alias for event: Successfully updated path alias for 72 out of 72 entities.

File: docroot/modules/custom/va_gov_backend/va_gov_backend.install (hunk 1 above)
Comment (oksana-c) [reply docroot/modules/custom/va_gov_backend/va_gov_backend.install]: when Kevin and I talked about aliases, we knew there's likely something that we may have missed. This is exactly the kind of feedback I needed. I'll reach out to connect on GovDelivery separately. 

There are only 2 banners on prod. It is not necessary to update aliases for them. I'll make that adjustment.

File: docroot/modules/custom/va_gov_backend/va_gov_backend.install (hunk 2 above)
Comment (swirtSJW) [reply docroot/modules/custom/va_gov_backend/va_gov_backend.install]: Awesome thank you.

**Pull request merged**